R/zoop_downloader.R

```diff
@@ -515,12 +516,15 @@ Zoopdownloader <- function(
                                                                        NA_character_,
                                                                        paste0(.data$SampleDate, " ", lubridate::hour(.data$TowTime), ":", lubridate::minute(.data$TowTime))),
                                                         "%Y-%m-%d %H:%M", tz="America/Los_Angeles"))%>%
+      #turbidity is now eitehr NTU or FNU
       tidyr::pivot_longer(cols=c(-"SampleDate", -"Survey", -"Station", -"TowTime", -"Temp", -"TopEC",
-                                 -"BottomEC", -"Secchi", -"Turbidity", -"Tide", -"BottomDepth", -"Duration", -"MeterCheck", -"Volume",
+                                 -"BottomEC", -"Secchi", -"NTU", -"FNU", -"Tide", -"BottomDepth", -"Duration", -"MeterCheck", -"Volume",
                                  -"Dilution", -"SampleID", -"Datetime"),
                           names_to="twentymm_Meso", values_to="CPUE")%>% #transform from wide to long
+      dplyr::mutate(Turbidity = dplyr::coalesce(NTU, FNU)) %>% #NTU and FNU are close enough it's not a big deal to treat them as the same
```

Comment:
I've heard wildly different opinions on this. For the water quality and fish integrated datasets, we've moved to separate columns for NTU and FNU ("TurbidityFNU", "TurbidityNTU"). Maybe we should do the same with zooper?

Reply:
Ugh. Sigh. I suppose you're probably right. I can make that change.

Reply:
Sorry. I can make the change for the other surveys if it's too much trouble

Reply:
Naw, I got it! All the other surveys use NTU, so it's just renaming them. I'm already halfway through testing the update.